### PR TITLE
Initial bylaws commit

### PR DESCRIPTION
**Pull request by ElijahCaine**
These rules (and the ideas of rules) spawned from a conversation in
\#osu-lug on January 22, 2015 about how there should only be one bot
in the channel at any time.

rules.rst is are community rules which are  enforced, created, modified,
etc by the LUG community for the LUG community. They are not bylaws nor
are they part of the constitution. They are guidelines set in something
between stone and passing comments.

Pull Requests Welcome.

### CODE REVIEW

File: rules.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Rules
+-----
+- There should be a maximum of one bot in the irc channel (#osu-lug,
+  irc.freenode.net) at any time. 
+
+Contributing
+------------
+How to propose a rule:
+0. Have a GitHub account.
```
Comment (mythmon) [inline rules.rst]: You should fix this so it shows up as an ordered list. I don't know how to do this with RST. /me pages @edunham.

File: rules.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+   master branch of the original repository. DO NOT MERGE THE PULL REQUEST YET.
+
+A pull request into master can only be merged with the approving comment (+1)
+of LUG two officers and six active LUG members. The final merge into master must
+be done by an officer of LUG.
+
+Any comment for or against the change in the comments of the pull request must
+include a reason for the support or non-support. Any -1 comments should either
+be fixed or addressed with a WONTFIX response. All comments must include the
+GitHub user's irc handle in #osu-lug.
```
Comment (mythmon) [inline rules.rst]: +"if different from their github handle"

File: rules.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+4. Create a Pull Request to merge your version of this repository into the
+   master branch of the original repository. DO NOT MERGE THE PULL REQUEST YET.
+
+A pull request into master can only be merged with the approving comment (+1)
+of LUG two officers and six active LUG members. The final merge into master must
+be done by an officer of LUG.
+
+Any comment for or against the change in the comments of the pull request must
+include a reason for the support or non-support. Any -1 comments should either
+be fixed or addressed with a WONTFIX response. All comments must include the
```
Comment (mythmon) [inline rules.rst]: Who gets to decide if it is addressed? Who gets to WONTFIX it?

File: rules.rst (hunk 1 above)
Comment (edunham) [inline rules.rst]: for reference, http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#enumerated-lists

```
1. thing one
2. thing two
```

File: rules.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
+LUG Rules
+=========
+
+Preamble
+--------
+This is a set of rules for the Oregon State University Linux Users Group. These
+are not set in stone and are in fact meant to be dynamic; adapting to the needs
+of the community as they change. These rules are seperate from our constitution
+(which is also in this repository) but ought to be enforced by the OSU Linux
+Users Groupcommunity.
```
Comment (lucywyman) [inline rules.rst]: Group community

File: rules.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+For information on contributing to the set of rules see the `Contributing`
+section below.
+
+Rules
+-----
+#. There should be a maximum of one bot in the irc channel (#osu-lug,
+   irc.freenode.net) at any time.
+#. Any malicious or (i.e., 'black hat') material shared in the official
```
Comment (lucywyman) [inline rules.rst]: s/or//

**Comment by edunham** [comment]
Looks like a good place to start for having actual rules. Probably a good time to role them out, also, considering that there don't seem to be any ongoing conflicts that would be changed by the rules' implementation at the moment. 

My one nitpick is that they should probably be called bylaws rather than just rules. This is because bylaws are enforced[*] by the constitution (https://github.com/OSULUG/laws/blob/master/constitution.rst#bylaws-2), which is what you submit every time you want to be re-recognized[*] as an Official OSU Student Group[*], which I assume would put the authority[*] of OSU on LUG's side if a major conflict over the rules was to occur. 

[*]: The correct legal terms for these things may be English synonyms of the words I've used

**Comment by ElijahCaine** [comment]
@edunham I agree that they should be called bylaws, my one concern is that I am concerned that this is not formal enough to be considered a set of bylaws. The constitution doesn't really lay out how to add a bylaw specifically.

That said, if the constitution doesn't say one way or another I have no opposition to making this the official bylaws and process for adding bylaws. It's just a concern.

@lucywyman @mythmon ping for thoughts if you've got them.

**Comment by edunham** [comment]
@ElijahCaine this document bootstraps a bylaw modification process, so we really only have to worry about the "legality" of getting it instated. You can probably assume that "fix gross oversights in the constitution" is a "duty not otherwise assigned" in your job description, https://github.com/OSULUG/laws/blob/master/constitution.rst#president-71 , and use your authority as president to merge things that the group agrees are desired and sensible.

**Comment by lucywyman** [comment]
+1 from me -- I have no strong opinions on whether the word bylaws should or should not be used, but fall on the side of bylaws should be the specific word used, just like this puppy

https://youtu.be/QxLHhNaMitg

**Pull request merged**